### PR TITLE
got rid of excess buttons and extra styles

### PR DESCRIPTION
**Pull request by Brixsta**

### CODE REVIEW

File: client/components/App.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ class App extends Component {
   }
 
   setCurrentItem(item_id){
-    // Updates new item (if user clicks different item, for example)
+    // Updates new item (if user clicks different i4tem, for example)
```
Comment (jolozano) [inline client/components/App.js:28]: is line 28 a typo? is it supposed to be 'i4tem' its a comment so its not a big deal.

**Review by jolozano — APPROVED** [review]
Everything looks good to me. We can go back and change the typo.  I saw where you added the eBay logo on the Search component. I didn't see conflicts. I am sending it.

**Comment by jolozano** [comment]
sent

**Comment by ComplexChris** [comment]
Great work, Charles, it is definitely rare to find developers that take time to clean/polish up code and to remove/refactor unnecessary code blocks (JS and CSS). You're killing it on this project, keep this up and throw in some more techniques you have not used before or not much of and you will be golden.

**Pull request merged**